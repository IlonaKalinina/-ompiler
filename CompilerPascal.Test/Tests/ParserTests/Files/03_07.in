program Test_03_07;
var
i: integer = 5;
begin
repeat
write(i);
i -= 1;
until i > 0;
end.