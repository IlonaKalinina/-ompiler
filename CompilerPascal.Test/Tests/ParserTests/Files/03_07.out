program Test_03_07
├───var
│    └─── :
│         ├─── i
│         ├─── integer
│         └─── =
│              └─── 5
│    
begin
├─── repeat
│    ├─── write
│    │    └─── i
│    ├─── -=
│    │    ├─── i
│    │    └─── 1
│    └─── >
│         ├─── i
│         └─── 0
└─── end
